bundle-size/package.json

```diff
@@ -17,7 +17,7 @@
     "deploy": "gcloud --project amp-bundle-size-bot app deploy",
     "setup-db": "node ./setup-db.js",
     "start": "probot run ./app.js",
-    "test": "jest"
+    "test": "LOG_LEVEL=warn jest"
```

Comment:
That's not good, this means if I run `npm test` locally it will ignore the value from the environment. Remove this and set the LOG_LEVEL in Travis

Reply:
Ahh good to know, done.